### PR TITLE
new addon purchase_requisition_auto_rfq_bid_selection

### PR DESCRIPTION
**Pull request by gurneyalex**
bridge module between purchase_requisition_auto_rfq and
purchase_requisition_bid_selection

### CODE REVIEW

**Comment by gurneyalex** [comment]
This PR depends on
- [x] #26
- [x] #28

**Comment by yvaucher** [comment]
@gurneyalex in tests kitchenset reference is missing

Is a demo data file missing ?

File: purchase_requisition_auto_rfq_bid_selection/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{"name": "Purchase Requisition Auto RFQ / Bid Selection bridge",
+ "summary": "Bridge module for PR Auto RFQ / Bid Selection",
+ "version": "0.1",
+ "author": "Camptocamp",
+ "category": "Purchase Management",
+ "license": "AGPL-3",
+ 'complexity': "normal",
+ "images": [],
+ "description": """Compatibility module for purchase_requisition_auto_rfq and
+    purchase_requisition_bid_selection.
```
Comment (lepistone) [inline purchase_requisition_auto_rfq_bid_selection/__openerp__.py]: I suggest to add something like:

Specifically, purchase_requisition_auto_rfq allows to create automatically RFQs for a tender, purchase_requisition_bid_selection distinguishes between bids and rfqs. This bridge module takes care that automatically generated documents are either bids of RFQs depending on the mode of the tender.

**Comment by lepistone** [comment]
I suggested a new paragraph for the module description. I can add that myself while merging.

Otherwise :+1: thanks.

**Comment by yvaucher** [comment]
:+1:

**Comment by gurneyalex** [comment]
@lepistone change approved

**Pull request merged**